configs/Lenovo/ThinkPad-X1-Carbon.conf

```diff
@@ -0,0 +1,20 @@
+# Tested with Lenovo ThinkPad X1 Carbon Gen 6
+
+chip "thinkpad-isa-0000"
+    ignore temp2
+    ignore temp3
+    ignore temp4
+    ignore temp5
+    ignore temp6
+    ignore temp7
+    ignore temp8
+    ignore temp9
+    ignore temp10
+    ignore temp12
+    ignore temp13
+    ignore temp14
+    ignore temp15
+    ignore temp16
```

Comment:
Please add a comment explaining why you're ignoring all these temperatures. Are they always zero? Do they report nonsensical values? That's the kind of info that I'm looking for.

Other than that, it looks good to me. Thanks!

Reply:
Documented as always zero.